ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/CoupledForm.java

```diff
@@ -0,0 +1,113 @@
+package io.syndesis.qe.fragments.common.form;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CoupledForm extends Form{
+
+    private String[] selectValues = new String[] {"yes","checked","check","select", "selected"};
```

Comment:
This String[] array can be List<String> probably as in only usage to check with `contains()` method.

Reply:
If it's Array I can use Arrays.toString(array) in a log message then. That's why I have used this.

Reply:
And there's similar [way] for collections also (https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-) `String.join("<delimiter>", list)`